### ISSUE TITLE
Five day minute data comes back at 10 minute intervals.

### ISSUE THREAD

**Issue by SuperDaveOsbourne** [vslee/IEXSharp#89]
var response = await iexCloudClient.StockPrices.HistoricalPriceAsync("IBM", ChartRange.FiveDayMinute);

12/21/2020 3:20:00 PM Open: 123.24, High: 123.45, Low: 123.235, Close: 123.45, Vol: 2687
12/21/2020 3:30:00 PM Open: 123.425, High: 123.47, Low: 123.12, Close: 123.34, Vol: 6532
12/21/2020 3:40:00 PM Open: 123.43, High: 123.43, Low: 123.13, Close: 123.21, Vol: 5257
12/21/2020 3:50:00 PM Open: 123.2, High: 123.54, Low: 122.985, Close: 123.38, Vol: 20583

Is this an artifact of my free key from them or another issue in the library? I am really looking for 15 minute intervals. I looked through the tests and didn't see anything that stood out to specify bar duration.

Seems to work fine in .Net 5 other than that.

**Comment by SuperDaveOsbourne** [comment]
Thanks, found it after I rebooted. Still some issues with 16.9 preview 2 of VS wasn't showing some of the tests where I found StockPrices.IntradayPricesAsync. I see a lot of missing time frames though. Is this because the reported data is only what trades on their system? From DDD today. 

12/22/2020 2:01:00 PM Open: 11.705, High: 11.71, Low: 11.705, Close: 11.71, Vol: 69
12/22/2020 2:02:00 PM Open: 11.705, High: 11.71, Low: 11.705, Close: 11.71, Vol: 70
12/22/2020 2:03:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:04:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:05:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:06:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:07:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:08:00 PM Open: 11.705, High: 11.71, Low: 11.705, Close: 11.71, Vol: 117
12/22/2020 2:09:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:10:00 PM Open: , High: , Low: , Close: , Vol: 0
12/22/2020 2:11:00 PM Open: 11.71, High: 11.71, Low: 11.71, Close: 11.71, Vol: 1
12/22/2020 2:12:00 PM Open: 11.705, High: 11.705, Low: 11.705, Close: 11.705, Vol: 330
12/22/2020 2:13:00 PM Open: 11.715, High: 11.715, Low: 11.715, Close: 11.715, Vol: 300
12/22/2020 2:14:00 PM Open: 11.715, High: 11.715, Low: 11.71, Close: 11.71, Vol: 21
